test/com/ableton/VirtualEnvTest.groovy

```diff
@@ -33,16 +33,13 @@ class VirtualEnvTest extends BasePipelineTest {
 
   @Test
   void cleanup() {
-    script.env['TEMP'] = 'C:\\Users\\whatever\\AppData\\Temp'
-
-    VirtualEnv venv = new VirtualEnv(script, 'python3.6')
+    VirtualEnv venv = new VirtualEnv(script)
```

Comment:
Removing the `python` argument belongs to e5190442246eea104e15ffa4a9f5d7648f203cc1, not  945f26ff9b108991a3c67cc1332143ab35997711.

Reply:
This was a rebase error, I think. See 11d3416.